plugins/orchestrator-backend/src/provider/OrchestratorEntityProvider.ts

```diff
@@ -164,8 +164,8 @@ export class OrchestratorEntityProvider
         kind: 'Template',
         metadata: {
           name: sanitizedId,
-          title: i.definition.name,
-          description: i.definition.description,
+          title: i.definition?.name,
```

Comment:
Does it make sense `definition` being `undefined` here?
If so, maybe we need to filter them out before `map`.